temporal-sdk/src/main/java/io/temporal/common/converter/DefaultDataConverter.java

```diff
@@ -91,9 +101,14 @@ public static DefaultDataConverter newDefaultInstance() {
    * Creates instance from ordered array of converters. When converting an object to payload the
    * array of converters is iterated from the beginning until one of the converters successfully
    * converts the value.
+   *
+   * <p>The newly created instance will use the default failure converter, configured to not encode
+   * message and stack trace. Call {@code withFailureConverterOverride} on the instance to set a
```

Comment:
You may prefer `@link` over `@code` especially when you are linking inside the same class and don't try express "calling a method on a VARIABLE". This will make it much nicer for users to travel around the javadocs, as it will be clickable.